### ISSUE TITLE
IMPORTANT - indentation

### ISSUE THREAD

**Issue by franfranfranfran** [grandpoisson/f1-3-c2p1-colmar-academy#3]
https://github.com/grandpoisson/f1-3-c2p1-colmar-academy/blob/master/Colmar-Academy-Submitted/index.html#L215-L224

try to keep your indentation / spacing as consistent as possible throughout your code.  a line being indented should indicate that it is nested inside something else and everything nested to the same level should be indented the same amount. your footer is a great example of a well-indented block of code. heres some more info about html styling in general - https://www.w3schools.com/html/html5_syntax.asp

**Comment by grandpoisson** [comment]
Yeah, this has been a recurring comment. I use sublime text and sadly it has no indent feature that I have found
